lib/Transforms/Instrumentation/ShadowMemSeaDsa.cc

```diff
@@ -163,6 +172,13 @@ class LocalAAResultsWrapper {
 };
 
 class ShadowDsaImpl : public InstVisitor<ShadowDsaImpl> {
+public:
+  using NodeIdMap = llvm::DenseMap<const sea_dsa::Node *, unsigned>;
+
+  /// \brief A map from DsaNode to its numeric id
+  NodeIdMap m_nodeIds;
```

Comment:
`m_nodeIds` should be private.

I am not sure why `NodeIdMap` type should be public either